calico-enterprise/reference/architecture/overview.mdx

```diff
@@ -102,9 +102,9 @@ Provides the API for listing, downloading, and rendering reports, and RBAC by pe
 
 A daemonset that runs checks in the CIS Kubernetes Benchmark on each node so you can see if Kubernetes is securely deployed.
 
-### Elasticsearch gateway
+### Datastore API
 
-**Main task**: Protects managed clusters from potential attacks in a multi-cluster management (mcm) deployment. Enforces that 1) managed cluster components do not get credentials for an Elasticsearch user (for access/permissions to Elasticsearch and Kibana), and 2) all Elasticsearch requests from a managed cluster are only on an index that the cluster is allowed to access. In short, a compromised component in a managed cluster cannot write data or create indexes for another cluster/tenant.
+**Main task**: A {{prodname}} HTTP API that enforces tenancy boundaries and  prerforms RBAC checks for added security. Protects managed clusters from potential attacks in a multi-cluster management (mcm) deployment. Enforces that 1) managed cluster components do not get credentials for an Elasticsearch user (for access/permissions to Elasticsearch and Kibana), and 2) all Elasticsearch requests from a managed cluster are only on an index that the cluster is allowed to access. In short, a compromised component in a managed cluster cannot write data or create indexes for another cluster/tenant.
```

Comment:
There is a lot of emphasis on MCM, but we are also protecting any access to Elastic as a data store.

Reply:
Rewrote to include comment.